### PR TITLE
Expand the view panel in stock operation

### PR DESCRIPTION
**Pull request by gabriel090**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

Expand the view panel in the stock management system
Modified UI

https://github.com/user-attachments/assets/31edc660-4d89-4d41-8315-afaa38e96a4f

### CODE REVIEW

**Comment by chimanwadike** [comment]
Thanks for this PR @gabriel090
It goes a long way to include JIRA ticket/issue number on PRs as well as adding link to the related issue in the PR summary.

Also, @gabriel090 you probably want to do git rebase instead of git merge. as merging would make it difficult to see your actual work by introducing lots of unrelated commits.

I hope this helps https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477236/Using+Git

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -10,8 +10,8 @@ on:
       - created
 
 env:
-  ESM_NAME: "@openmrs/esm-stock-management-app"
-  JS_NAME: "openmrs-esm-stock-management-app.js"
+  ESM_NAME: "@kenyaemr/esm-stock-management-app"
```
Comment (jabahum) [inline .github/workflows/node.js.yml:13]: do we really need an implementation specific module

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-    "name": "@openmrs/esm-stock-management-app",
-    "version": "1.0.0",
+    "name": "@kenyaemr/esm-stock-management-app",
```
Comment (jabahum) [inline package.json:2]: change this too

**Review by jabahum — CHANGES_REQUESTED** [review]
Please resolve conflicts and seems this PR is bringing back really old changes

**Pull request closed without merging**